beancount/utils/misc_utils_test.py

```diff
@@ -234,10 +234,8 @@ def test_is_sorted(self):
         self.assertTrue(misc_utils.is_sorted([7, 6, 3, 1], cmp=operator.gt))
 
     def test_escape_string(self):
-        assert (
-            misc_utils.escape_string('Entry with escaped "symbols" \\ \r \n')
-            == 'Entry with escaped \\"symbols\\" \\\\ \r \n'
-        )
+        escaped = misc_utils.escape_string('Entry with escaped "symbols" \\')
```

Comment:
Why remote \r and \n from this test?

Reply:
Because they are not escaped anyway and their presence seems to trip up readers of this code. I'll put them back and add a comment instead.